modules/cnf-installing-the-n3000-operator.adoc

```diff
@@ -0,0 +1,157 @@
+
+For vRAN use cases the OpenNESS Operator for Intel® FPGA PAC N3000 is used alongside the OpenNESS Operator for Intel® Wireless FEC Accelerator.
+
+As a cluster administrator, you can install the OpenNESS Operator for Intel® FPGA PAC N3000 by using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the Operator by using the CLI
+
+As a cluster administrator, you can install the Operator by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the N3000 Operator by completing the following actions:
+
+.. Define the `vran-acceleration-operators` namespace by creating a file named `n3000-namespace.yaml` file as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: vran-acceleration-operators
+    labels:
+        openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-namespace.yaml
+----
+
+. Install the N3000 Operator in the namespace you created in the previous step:
+
+.. Create the following `OperatorGroup` CR and save the YAML in the `n3000-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: n3000-operators
+    namespace: vran-acceleration-operators
+spec:
+    targetNamespaces:
+    - vran-acceleration-operators
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-operatorgroup.yaml
+----
+
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest n3000-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+[source,terminal]
+----
+stable
+----
+
+.. Create the following `Subscription` CR and save the YAML in the `n3000-sub.yaml` file:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: n3000-subscription
+    namespace: vran-acceleration-operators
+spec:
+    channel: "<channel>"        <1>
+    name: n3000
+    source: certified-operators <2>
+    sourceNamespace: openshift-marketplace
+----
+<1> Specify the value for channel from the value obtained in the previous step for the `.status.defaultChannel` parameter.
+<2> You must specify the `certified-operators` value.
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-sub.yaml
+----
+
+.. Change to the `vran-acceleration-operators` project:
++
+[source,terminal]
+----
+$ oc project vran-acceleration-operators
+
```

Comment:
Please remove blank line.

Reply:
I get your point when I started this write up I was following the guidance in the u/s docs which indicated this is the way it is done. I guess I was ensuring steps broadly followed that described upstream